### PR TITLE
Yin2020center based 1 marcobueso

### PR DESCRIPTION
**Pull request by marcobueso**
Hi @SShivang, ready for initial review! I apologize for submitting a bit late.

### CODE REVIEW

File: _summaries/yin2020center-based_1.md [hunk 1]
```diff
@@ -3,17 +3,43 @@ layout: summary
...
+Tables 1 and 2 show results obtaiend for 3D detection on Waymo test set, and nuScenes test set. In both, and other tests not shown below, CenterPoint performs better than all other models.\
+![Yin (2020).](yin2020_center_based_1b.PNG)
+
+![Yin (2020).](yin2020_center_based_1c.PNG)
+
+Authors also show that center-based methods for 3D detection, like CenterPoint, outperform anchor-based methods.
+
+
+### What interesting variants are explored?
+Initial ablation study was center-based v. anchor-based methods. As mentioned above, center-based performs better.\
```
Comment (SShivang) [inline _summaries/yin2020center-based_1.md:38]: Maybe you could include the figure from the paper regarding the ablation, and discuss the results of the ablation.

File: _summaries/yin2020center-based_1.md [hunk 2]
```diff
@@ -3,17 +3,43 @@ layout: summary
...
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+### What is the core idea?
+The authors propose CenterPoint, a 3D object-tracking approach which simplifies the algorithm to a greedy closest-point matching. \
+The paper aims to represent objects as points, avoiding difficulties on other models, related to the object's orientation, rotation, etc.
+
```
Comment (SShivang) [inline _summaries/yin2020center-based_1.md:13]: I think you can also add why this method works better than previous works.

File: _summaries/yin2020center-based_1.md [hunk 3]
```diff
@@ -3,17 +3,43 @@ layout: summary
...
+
+
+### How is it realized (technically)?
+CenterPoint uses a standard Lidar-based backbone network, (VoxelNet or PointPillars), to build a representation of the input point-cloud. It flattens the point-cloud and maps the center of the object. Then, it builds the 3D features of the object based on the found centerpoint.\
```
Comment (SShivang) [inline _summaries/yin2020center-based_1.md:17]: Good explanation.

File: _summaries/yin2020center-based_1.md [hunk 4]
```diff
@@ -3,17 +3,43 @@ layout: summary
...
+
+### How well does the paper perform?
+The paper tests the models on two popular large datasets: Waymo Open Dataset, and nuScenes Dataset.\
+The testing shows a 3-4 mAP increase in 3D detection under different backbones. With some additional overhead (10%), a two-stage refinement inceases mAP by 2 more points.
+Tables 1 and 2 show results obtaiend for 3D detection on Waymo test set, and nuScenes test set. In both, and other tests not shown below, CenterPoint performs better than all other models.\
+![Yin (2020).](yin2020_center_based_1b.PNG)
+
+![Yin (2020).](yin2020_center_based_1c.PNG)
+
+Authors also show that center-based methods for 3D detection, like CenterPoint, outperform anchor-based methods.
```
Comment (SShivang) [inline _summaries/yin2020center-based_1.md:34]: Again, could mention why they outperform. What is the factor that the other models didn't have?

**Review by SShivang — CHANGES_REQUESTED** [review]

**Comment by marcobueso** [comment]
Comments addressed and ready for submission.

**Review by zhaoyue-zephyrus — APPROVED** [review]

**Pull request merged**